### ISSUE TITLE
Example

### ISSUE THREAD

**Issue by johntom** [bryanrsmith/aurelia-angular-loader#1]
Hi Bryan,
I'm intrigued by this idea but I'm not sure how to implement with Aurelia.I'd love to make a showcase to demonstrate at my talk at the next http://www.meetup.com/ng-nyc/. 
I have a couple of directives that would be great to implement.
angular-file-upload ("https://github.com/nervgh/angular-file-upload") and angular-xeditable ( "http://vitalets.github.io/angular-xeditable") I'd be happy to post repo as an example.
Thanks,
John

**Comment by bryanrsmith** [comment]
Hi, @johntom.

I wrote up a better readme and totally forgot to push it. Sorry about that! It's up now-- give it a skim & let me know if I can clarify anything.

https://github.com/bryanrsmith/aurelia-angular-loader/blob/master/README.md

As noted in the readme, there are currently some limitations on the directives that can be used. Attribute directives can't currently be used because I have to inject an element to use as the Angular application root. The element that the directive runs on is not the element in the Aurelia view; it's a generated element. Because of this I restricted it to 'E' Angular directives.

I also notice that the Angular file upload directives use `scope.$eval(attributes.uploader)` to retrieve an uploader instance. This probably won't work right because a new scope is created for each directive (since each directive instance is in a new Angular app). Currently the only way to pass data between the two frameworks is to connect Aurelia bindings to Angular isolate scope properties. I might be able to make the Angular scope inherit Aurelia's binding context to improve this...
